documentation/asciidoc/topics/ref_xsite_crd_automatic.adoc

```diff
@@ -16,9 +16,9 @@ ifndef::productized[]
 <4> Provides connection information for all backup locations.
 <5> Specifies a backup location that matches `.spec.service.sites.local.name`.
 <6> Specifies a backup location.
-//* Use `xsite://` if the backup location has a static hostname and port.
 * Use `minikube://` if the backup location is a Minikube instance.
```

Comment:
kubernetes instead of minikube?

Reply:
Yes, as minikube just one of the many possible deployment of the k8s

Reply:
why special-case minikube then ? Unless it requires special handling it should just fall under the `kubernetes` case.

Reply:
Yes, @tristantarrant will change it to `kubernetes` in sepearte PR #764